deepinv/datasets/div2k.py

```diff
@@ -57,16 +59,20 @@ class DIV2K(torch.utils.data.Dataset):
     """
 
     # https://data.vision.ee.ethz.ch/cvl/DIV2K/
-    archive_urls = {
-        "DIV2K_train_HR.zip": "http://data.vision.ee.ethz.ch/cvl/DIV2K/DIV2K_train_HR.zip",
-        "DIV2K_valid_HR.zip": "http://data.vision.ee.ethz.ch/cvl/DIV2K/DIV2K_valid_HR.zip",
-    }
+    archive_urls = MappingProxyType(
+        {
+            "DIV2K_train_HR.zip": "http://data.vision.ee.ethz.ch/cvl/DIV2K/DIV2K_train_HR.zip",
+            "DIV2K_valid_HR.zip": "http://data.vision.ee.ethz.ch/cvl/DIV2K/DIV2K_valid_HR.zip",
+        }
+    )
```

Comment:
Set it to private as well?

```suggestion
    _archive_urls = MappingProxyType(
        {
            "DIV2K_train_HR.zip": "http://data.vision.ee.ethz.ch/cvl/DIV2K/DIV2K_train_HR.zip",
            "DIV2K_valid_HR.zip": "http://data.vision.ee.ethz.ch/cvl/DIV2K/DIV2K_valid_HR.zip",
        }
    )
```